tests/fixtures/simple-failing-dockerfile/Dockerfile

```diff
@@ -2,6 +2,5 @@ FROM busybox:1.31.0-uclibc
 LABEL com.docker.compose.test_image=true
 LABEL com.docker.compose.test_failing_image=true
 # With the following label the container wil be cleaned up automatically
-# Must be kept in sync with LABEL_PROJECT from compose/const.py
```

Comment:
I didn't understand this removal

Reply:
PyCharm went confused during refactoring it seems :P